test/test_cli_integration.py

```diff
@@ -130,21 +153,21 @@ def setup_cli_params(self, env_vars, *params):
 
     def setUp(self):
         self.setup_cli_params({})
-        return invoke_cli(self, self.cli_auth_params,
-            ['policy', 'replace', 'root', 'test/test_config/initial_policy.yml'])
+        return self.invoke_cli(self.cli_auth_params,
+                               ['policy', 'replace', 'root', self.environment.path_provider.get_policy_path(
+                                   "initial")])  # 'test/test_config/initial_policy.yml'
 
     def set_variable(self, variable_id, value, exit_code=0):
-        return invoke_cli(self, self.cli_auth_params,
-            ['variable', 'set', variable_id, value], exit_code=exit_code)
-
+        return self.invoke_cli(self.cli_auth_params,
+                               ['variable', 'set', variable_id, value], exit_code=exit_code)
 
     def initalize_cli(self, variable_id, value, exit_code=0):
-        return invoke_cli(self, self.cli_auth_params,
-            ['init', '--url', 'https://localhost-https', '--account', 'dev'], exit_code=exit_code)
+        return self.invoke_cli(self.cli_auth_params,
+                               ['init', '--url', 'https://localhost-https', '--account', 'dev'], exit_code=exit_code)
```

Comment:
i believe this needs to be changed from https://localhost-https to something else?

Reply:
indeed. I changed into self.client_params.hostname